README.md

```diff
@@ -1,3 +1,6 @@
+<img src="assets/logo/horizontal/color/krew-horizontal-color.png" width="480"
+  alt="Krew logo"/>
+
 # krew
```

Comment:
I wonder if we still need the heading, if the logo already says "krew".

Reply:
That's actually still useful for SEO, since it's an h1.